api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -2904,6 +2918,161 @@ components:
               description: Note on this new user.
               type: string
 
+    publicCategoryListResponse:
+      description: JSON data describing the public category response
+      type: object
+      properties:
+        _links:
+          type: object
+          properties:
+            self:
+              type: object
+              properties:
+                href:
+                  type: string
+                  description: URI to the list endpoint
+                  format: uri
+            next:
+              type: object
+              properties:
+                href:
+                  type: string
+                  description: URI to the next page of the list endpoint (if not available this is null)
+                  format: uri
+            previous:
+              type: object
+              properties:
+                href:
+                  type: string
+                  description: URI to the previous page of the list endpoint (if not available this is null)
+                  format: uri
+        count:
+          type: integer
+          description: Total count of results for the request
+        results:
+          type: array
+          description: A list of categories, paginated
+          items:
+            $ref: '#/components/schemas/publicCategoryParentResponse'
+
+    publicCategoryParentResponse:
+      description: JSON data describing the public parent category response
+      type: object
+      properties:
+        _links:
+          type: object
+          properties:
+            curies:
+              type: object
+              properties:
+                name:
+                  type: string
+                  description: The compact URI (curie) named 'sia'
+                href:
+                  type: string
+                  format: uri
+            self:
+              type: object
+              properties:
+                href:
+                  type: string
+                  description: URI to the public category
+                  format: uri
+        _display:
+          type: string
+          description: The display value of the category
+          example: Afval
+        name:
+          type: string
+          description: The name of the category
+          example: Afval
+        slug:
+          type: string
+          description: The (generated) slug of the category
+          example: afval
+        public_name:
```

Comment:
I have to comment that we now have three name like fields here `_display`, `name` and `public_name`. We must add some documentation what each is used for - because as is, the presence of three of these fields is confusing.

We seem to be adding the `public_name` for the map, but is it also intended for use elsewhere?

Reply:
`_display` is a `HAL` standard field. It is not saved in the database. The `name` and the `public_name` will be saved in the database